kubernetes/deployment-staging.tmpl

```diff
@@ -288,6 +203,107 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  name: panoptes-staging-sidekiq
+  labels:
+    app: panoptes-staging-sidekiq
+spec:
+  selector:
+    matchLabels:
+      app: panoptes-staging-sidekiq
+  template:
+    metadata:
+      labels:
+        app: panoptes-staging-sidekiq
+    spec:
+      containers:
+        - name: panoptes-staging-sidekiq
+          image: zooniverse/panoptes:__IMAGE_TAG__
+          resources:
+            requests:
+              memory: "700Mi"
+              cpu: "50m"
+            limits:
+              memory: "700Mi"
+              cpu: "1000m"
+          args: ["/rails_app/scripts/docker/start-sidekiq.sh"]
+          envFrom:
+          - secretRef:
+              name: panoptes-common-env-vars
+          - secretRef:
+              name: panoptes-staging-env-vars
+          - configMapRef:
+              name: panoptes-staging-shared
+          volumeMounts:
+          - mountPath: /tmp
+            name: panoptes-staging-dumpworker-data
+      volumes:
+        - name: panoptes-staging-dumpworker-data
+          emptyDir: {}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: panoptes-staging-redis
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: azurefile
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: panoptes-staging-redis
+  labels:
+    app: panoptes-staging-redis
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: panoptes-staging-redis
+  template:
+    metadata:
+      labels:
+        app: panoptes-staging-redis
+    spec:
+      containers:
+        - name: panoptes-staging-redis
+          image: redis:6.0.5
```

Comment:
https://github.com/zooniverse/operations/blob/3e30539794fd4983e6be7a0726f8b763b280ef54/panoptes-redis/docker-compose.yml#L3

will this image need to add a start script for running with persistence? https://hub.docker.com/_/redis/
```
start with persistent storage

$ docker run --name some-redis -d redis redis-server --appendonly yes

If persistence is enabled, data is stored in the VOLUME /data, which can be used with --volumes-from some-volume-container or -v /docker/host/dir:/data (see docs.docker volumes).
```
For more about Redis Persistence, see http://redis.io/topics/persistence.

And maybe a custom conf? https://github.com/zooniverse/operations/blob/3e30539794fd4983e6be7a0726f8b763b280ef54/panoptes-redis/docker-compose.yml#L8